packages/fluent-theme/src/fluent/styles/Rating.styles.ts

```diff
@@ -0,0 +1,88 @@
+import { IRatingStyleProps } from 'office-ui-fabric-react/lib/Rating';
+import { NeutralColors } from '../FluentColors';
+
+export const RatingStyles = (props: IRatingStyleProps) => {
+  const { disabled, readOnly, theme } = props;
+  const { semanticColors } = theme;
+
+  // Copied over from the source default styles
```

Comment:
I'm not sure about copying over the entire `Rating.styles.ts` file instead of applying targeted overrides as we've been doing. It's true that this will make it easier to merge the Fluent styles into the default at some point (it becomes more of a swap than a merge) but it's difficult to tell what the actual Fluent changes are. It also means that if any style fixes are made to the default theme, we have to copy them into the Fluent theme.

Reply:
Nevermind, that's not what you were doing here. 😆 I saw so much code and assumed that it was a copy (like we had discussed yesterday as a potential way to do this) but now that I'm looking at the default theme I see that it's not. There are just more changes here than in most components.

Reply:
It seems like many of these changes should be made to the default theme. In Fluent the only change is the colors, right? @betrue-final-final

Reply:
We also want to move towards style variables like [the button styleVariables shown here](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/theme-samples/src/WordCustomizations.ts). This approach is both simpler and less implementation dependent (i.e. you don't have to know all of the selectors and internals of each component section.)

Reply:
@JasonGore Thanks for pointing that out, I didn't realize we'd started to implement that idea.

Reply:
@mikewheaton 
Yes... there was no intention to copy the styles 😄 . The main reason I brought those constants in here was to have a clear visibility on height calculation below. Instead of having `height: 16 + ratingVerticalPadding * 2`, where you would wonder why `16`, I just copied them over and put a comment to indicate that this is the same with default and did not changed with fluent, so that the style would look `height: ratingSmallIconSize + ratingVerticalPadding * 2`

Also, I am not sure what are the specs in the default Fabric toolkit for Rating so I assumed that the ones checked in the master are the ones so when doing fluent updates I noticed the differences and implemented them in here. @betrue-final-final can you have a quick pass over the default (not fluent) toolkit and see if the Rating needs some style fixes. If yes this should be done as a separate issue.

@JasonGore 
I can't wait for us to move to `styleVariables` pattern as it would be infinitely easier to theme or custom style the components, **but I am confused on your comment**... is this something was discussed doing it along with the fluent effort? As @mikewheaton mentioned... I didn't realize we had started with the conversion...

Reply:
Sorry for introducing confusion. I just wanted to mention it as an FYI, there isn't much you can do until components are modified to support this approach. :)

Reply:
We'll keep going ahead with the current plan. The styles that we override in the Fluent theme will be good hints for which style variables would be useful on these components when the time comes. :)